### PR TITLE
Proofpoint tap improvements

### PR DESCRIPTION
**Pull request by anara123**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16919
fixes: https://github.com/demisto/etc/issues/15048

## Description
Integration was rewritten - V2

## Does it break backward compatibility?
   - No: it is not fully compatible with V1

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by anara123** [comment]
@kirbles19 please review and fix 
https://github.com/demisto/content/blob/proofpoint-tap-improvements/Integrations/ProofpointTAP_v2/ProofpointTAP_v2.yml

**Comment by kirbles19** [comment]
@anara123 .yml reviewed.

File: Integrations/ProofpointTAP_v2/ProofpointTAP_v2_description.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Create an API account
```
Comment (glicht) [inline Integrations/ProofpointTAP_v2/ProofpointTAP_v2_description.md]:
```suggestion
## Configure an API account
```

File: Releases/LatestRelease/ProofpointTAP_v2.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Proofpoint TAP integration rewrite
```
Comment (glicht) [inline Releases/LatestRelease/ProofpointTAP_v2.md]: More details. What was improved/changed?

File: Integrations/ProofpointTAP_v2/ProofpointTAP_v2.yml [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+description: Use the Proofpoint Targeted Attack Protection (TAP) integration to protect against and
+  provide additional visibility into phishing and other malicious email attacks.
+display: ProofpointTAP_v2
```
Comment (glicht) [inline Integrations/ProofpointTAP_v2/ProofpointTAP_v2.yml]:
```suggestion
display: Proofpoint TAP v2
```

**Comment by kirbles19** [comment]
Re-reviewed the .yml and made a few minor changes.

**Comment by glicht** [comment]
@kirbles19 take also a look at the release notes: 
https://github.com/demisto/content/edit/proofpoint-tap-improvements/Releases/LatestRelease/ProofpointTAP_v2.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F3834

**Comment by kirbles19** [comment]
RN updated.

**Comment by kirbles19** [comment]
We need to generate docs for this and post them here https://github.com/demisto/etc/issues/17545

**Pull request merged**